manifests/key.pp

```diff
@@ -15,7 +15,7 @@
     command     => "/usr/sbin/dnssec-keygen -a HMAC-MD5 -r /dev/urandom -b 128 -n USER ${name}",
     cwd         => "${cfg_dir}/bind.keys.d",
     require     => [
-      Package['dnssec-tools','bind9'],
+      Package['dnssec-tools',$package],
```

Comment:
I don't understand, where does this variable come from? I don't see it in the namespace?